scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
@bfredl It still does not tell whether helpers.lua is of “lua helpers.lua --do-something” type or a lua module. Separating shell commands in bin/ or scripts/ from the rest of the stuff is not uncommon and not unexpected, I do not see why that should be broken. generators/bin/ with generators/lib/ would be OK I think, I hope after some time generators/lib/ will contain templating engine and generators/templates/ will appear.